fulp_modules/features/exclusive_fauna/exiled_king_basic.dm

```diff
@@ -0,0 +1,702 @@
+#define BB_KING_PORTAL "BB_king_portal"
+#define BB_KING_TENTACLE "BB_king_tentacle"
+#define BB_KING_SURROUND_TENTACLE "BB_king_surround_tentacle"
+#define BB_CHARGE_ABILITY "BB_charge_ability"
+#define BOSS_MEDAL_EXILED "Exiled Killer"
+#define EXILED_KING_SCORE "Exiled Killed"
+#define BB_KING_TENTACLE_TRACK "BB_king_tentacle_track"
+
+/mob/living/basic/exiled_king
+	name = "Exiled King"
+	desc = "and what army?"
```

Comment:
```suggestion
	desc = "And what army?"
```
either that or make it a bit more objective like other fauna ("A monstrous creature protected by heavy shielding.")